src/pages/DataAccessRequestApplication.js

```diff
@@ -192,7 +192,7 @@ class DataAccessRequestApplication extends Component {
     }
     this.setState(prev => {
       prev.completed = completed;
-      prev.formData = formData;
+      prev.formData = fp.merge(prev.formData, formData);
```

Comment:
When creating a DAR from Dataset Catalog it generates json that doesn't have all of the keys initialized to empty strings (which the DARApplication does in its constructor). So when this init function was called, it would completely overwrite the initial blank values on the state values with undefined due to missing keys in the partial. Merging the payload into the state variable will ensure proper updates while maintaining the initialized blank values for the missing values.